scripts/example-pipelines-test.sh

```diff
@@ -3,6 +3,13 @@
 cp -L $KUBECONFIG /tmp/kubeconfig && export KUBECONFIG=/tmp/kubeconfig
 export IMG=${CI_OPERATOR_IMG}
 
+# switch to faster storage class for example pipelines tests (slower storage class is causing timeouts due 
+# to not able to copy whole windows disk)
+if [[ ! "$(oc get storageclass | grep -q 'ssd-csi (default)')" ]] && [[ $TARGET =~ windows.* ]]; then
```

Comment:
You could try to check if grep was successful / failed via exit code similar to here: https://github.com/lyarwood/common-instancetypes/blob/0a090f57672852fd5c4b4dd1d8302cf02d138101/scripts/check.sh#L4